src/editor/objects/Model.js

```diff
@@ -246,7 +246,7 @@ export default class Model extends Object3D {
     }
 
     this._src = source._src;
-    this.activeClipIndices = source.activeClipIndices || (source.activeClipIndex && [source.activeClipIndex]) || [];
+    this.activeClipIndices = source.activeClipIndices;
```

Comment:
Ok, I'm glad this line went back. This was the weirdest feeling change I made. (And I don't have a good grasp on this code yet)